src/t2iapi/device/device_responses.proto

```diff
@@ -15,6 +15,14 @@ import "t2iapi/biceps/metadata.proto";
 option java_package = "com.draeger.medical.t2iapi.device";
 option java_outer_classname = "DeviceResponses";
 
+/*
+Response containing all languages supported by the UI component represented by a MDS
```

Comment:
```suggestion
Response containing all languages supported by the UI component represented by an MDS.
```

Reply:
done

Reply:
Not really, you applied half of my change manually, it's missing the other half.